docs/pages/joy-ui/api/menu-item.json

```diff
@@ -1,37 +1,5 @@
 {
-  "props": {
```

Comment:
Is the removal of props intended here?

Reply:
It's a side-effect, but not an incorrect one. Joy's MenuItem doesn't define any own props and uses only ones inherited from ListItemButton.
While IMO it would be much clearer for the readers to have all the props on a single page, this is not in the scope of this PR.